packages/editor/src/components/post-publish-panel/postpublish.js

```diff
@@ -23,6 +23,27 @@ import { decodeEntities } from '@wordpress/html-entities';
  */
 import PostScheduleLabel from '../post-schedule/label';
 
+const POSTNAME = '%postname%';
+
+/**
+ * Returns real URL of a post. PermalinkTemplate holds all necessary
+ * information for the real URL.
+ *
+ * @param {Object} post         Post object.
+ *
+ * @return {string} PostPublish URL.
+ */
+
+const createPostURL = ( post ) => {
```

Comment:
Though a bit long, something like `getFuturePostUrl` might be clearer.

Reply:
fixed